Library/Formula/pam-u2f.rb

```diff
@@ -0,0 +1,41 @@
+class PamU2f < Formula
+  desc "Provides an easy way to use U2F-compliant authenticators with PAM."
+  homepage "https://developers.yubico.com/pam-u2f/"
+  url "https://github.com/Yubico/pam-u2f.git", :commit => "95fb0ec65475e8afa15aef1c14d1457d1f80fece"
```

Comment:
They have a prepackaged upstream release, but it doesn't compile on OS X.

FWIW, it's the same issue encountered in https://github.com/Homebrew/homebrew/issues/43565#event-428670625. See below:

```
$ make
Making all in .
  CC       pam-u2f.lo
  CC       util.lo
util.c:41:5: warning: implicit declaration of function 'close' is invalid in C99 [-Wimplicit-function-declaration]
    close(fd);
    ^
util.c:158:54: warning: format specifies type 'int' but the argument has type 'size_t' (aka 'unsigned long') [-Wformat]
          D(("Length of key number %d is %d", i + 1, devices[i].key_len));
                                         ~~          ^~~~~~~~~~~~~~~~~~
                                         %zu
./util.h:28:12: note: expanded from macro 'D'
    printf x;                                                         \
           ^
util.c:453:12: warning: initializing 'char *' with an expression of type 'const char *' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    .msg = prompt
           ^~~~~~
3 warnings generated.
  CCLD     pam_u2f.la
/bin/sh /Users/dan/Downloads/pam-u2f-pam_u2f-1.0.2/build-aux/missing a2x --format=manpage -a revdate="Version 1.0.2" man/pam_u2f.8.txt
a2x: ERROR: "xmllint" --nonet --noout --valid "/Users/dan/Downloads/pam-u2f-pam_u2f-1.0.2/man/pam_u2f.8.xml" returned non-zero exit status 4
make[1]: *** [man/pam_u2f.8] Error 1
make: *** [all-recursive] Error 1
```